src/caldav/init-database.sh

```
@@ -38,6 +38,14 @@ until pg_isready -q -h "$PGHOST" -p "$PGPORT" -U "$PGUSER"; do
   sleep 1
 done
 
+# Ensure the target database exists. Connect to the 'postgres' maintenance DB
+# and create PGDATABASE if missing. Safe to run repeatedly.
+if ! PGDATABASE=postgres psql -tAc \
```

Shellcheck (0.11.0)

[warning] This assignment is only seen by the forked process.

(SC2097)